src/sandstorm/run-bundle.c++

```diff
@@ -1370,6 +1372,31 @@ private:
     // in, copy its contents to /etc/resolv.conf.host-initial so we can use that if needed.
     backupResolvConf();
 
+    // Mount the cgroup2 filesystem at run/cgroup2:
```

Comment:
What happens if the host system is running cgroups v1? Can a namespace still use v2?

Reply:
You can have both mounted simultaneously, although they can't be managing the same controllers. So if v1 has already claimed the cpu, memory, etc controllers, then mounting here will still work, and we can still do things like free the cgroup, but we'll need to gracefully handle the case where controllers we want to use aren't available. But right now we don't try to use any of them anyway, so it's fine.